src/clr-angular/data/stack-view/stack-block.spec.ts

```diff
@@ -110,6 +110,96 @@ export default function(): void {
       expect(getBlockInstance(fixture).expandable).toBeTruthy();
     });
 
+    it('displays a caret when the stack block is expandable', () => {
+      fixture = TestBed.createComponent(DynamicBlock);
```

Comment:
Every unit test is creating its component, why don't we just move these two lines into `beforeEach()`? It doesn't look like it will require significant changes to do that.

Reply:
@Shijir: I noted the same exact point in the PR description. I didn't want to pollute this PR with other changes as we are trying to keep the PRs small. What do you think @youdz?

Reply:
Yes. I saw that. That's why I am not blocking this PR. But it seemed it won't pollute this PR much as it's evident that only two exact same lines are repeated in every single test.

Reply:
I think its more than that. We would have to break the tests into `View` and `Typescript API`. Then we would also have to create a separate file for Integration tests. If we do start to fix things, I would assume we would fix all the above things, no? I don't mind doing this with a separate PR but adding it in this PR is something that I wasn't sure about.

Reply:
As of now, I followed the same style that other tests in this spec follow.